deployment/ccip/changeset/testhelpers/messagingtest/helpers.go

```diff
@@ -20,7 +20,7 @@ import (
 	solconfig "github.com/smartcontractkit/chainlink-ccip/chains/solana/contracts/tests/config"
 	"github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings/v0_1_0/ccip_router"
 	solcommon "github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
-	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+	ccipocr3common "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
```

Comment:
Looks like an artifact from the migration, change it back?
```suggestion
	ccipocr3 "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
```